cmd/error.go

```diff
@@ -55,14 +55,24 @@ func (r *ReportError) Report() {
 }
 
 func errorQuit(err error) {
-	newReportError(err).Report()
+	currentUser, autherr := config.C.CurrentUser()
+	if autherr != nil {
+		debug.Println("Fail to get current user")
+		debug.Println(errgo.Details(err))
```

Comment:
```suggestion
		debug.Println(errgo.Details(err))
		return
```

Not sure whether you want to keep executing the function or not when you fail to get the user.

Reply:
Yes I'm sure it's just an HTTP Client finally, nothing has an internal state

Reply:
Your comment is related to another one of my comment. But you didn't addressed this one.

Reply:
Oh my bad this was about the Client and goroutines right

Reply:
Actually I want to keep on, at worse, the user will be nill for rollbar which is completely ok and taken into account